### PR TITLE
ci: Ensure the CI project always has a JWT signing key

### PR DESCRIPTION
**Pull request by jeremy-stytch**
The sandbox tests started failing in CI recently. The API was unable to sign the sandbox-session JWT for the CI project because the original signing key expired.

Normally, the key would automatically be rotated when it was necessary to sign the JWT for a real session. However, the CI project only ever issues sandbox requests and never starts a real session. So this uses the workaround of fetching the JWKS to trigger signing key generation outside the sandbox.

This is a _very_ specific edge case that would only be encountered by a very new or very old Stytch project whose first interaction with sessions is to try and validate the sandbox token. You should never have to use this anywhere in your real projects, Test or Live.

### CODE REVIEW

File: test/sandbox.test.ts [hunk 1]
```diff
@@ -138,7 +138,23 @@ describeIf(
...
-      test("success", () => {
+      test("success", async () => {
+        // Make sure there's a key that can be used to sign the sandbox JWT.
+        //
+        // You should never need to trigger this for a real project in either environment (Test or
+        // Live). Signing keys will be automatically generated for your project the first time the
+        // API needs one to sign a JWT. Those keys will be automatically rotated as you continue
+        // to use sessions.
+        //
+        // We only need this in this test suite because it's a very narrow edge case:
```
Comment (max-stytch) [inline test/sandbox.test.ts:149]: It might make a lot more sense for future readers to just write this as a test of `client.session.jwks` producing the intended result, instead of talking about the internals of Sandboxing

Comment (jeremy-stytch) [reply test/sandbox.test.ts]: That's a good point. I wanted it to block the call to `client.sessions.authenticate`, and I didn't know whether Jest would allow running tests in parallel. I was worried that someone would think they need this if they were using these tests as example code.

I was having trouble spelling the vague assert I need to do in callback form, but `async` made it readable again: a0527adddc4864646524874eaf4341a53731cbb7

Thanks!

**Review by max-stytch — APPROVED** [review]
